aries-site/src/layouts/navigation/SearchResults.js

```diff
@@ -0,0 +1,107 @@
+import { useEffect, useRef, useState } from 'react';
+import PropTypes from 'prop-types';
+import { Box, Button, Keyboard, List } from 'grommet';
+import { FormClose } from 'grommet-icons';
+
+import { SearchInput, SearchResult } from '.';
+
+export const SearchResults = ({
+  allSuggestions,
+  onChange,
+  onClose,
+  onEnter,
+  onSelect,
+  query,
+  results,
+  setSuggestions,
+  ...rest
+}) => {
+  const searchRef = useRef();
+  const step = 4;
+  const defaultPage = 1;
+  const [page, setPage] = useState(defaultPage);
+
+  useEffect(() => {
+    if (searchRef.current) {
+      searchRef.current.focus();
+    }
+  }, [searchRef]);
+
+  useEffect(() => {
+    setPage(defaultPage);
+  }, [query]);
+
+  return (
+    <Box
+      background="background-front"
+      round="small"
+      pad={{ horizontal: 'medium', top: 'small', bottom: 'large' }}
+      gap="small"
+      width="large"
+      {...rest}
+    >
+      <Button icon={<FormClose />} onClick={onClose} alignSelf="end" />
+      <Box
+        background="background-contrast"
+        flex={false}
+        round="xsmall"
+        margin={{ horizontal: 'small' }}
+      >
+        <Keyboard onEsc={onClose} onEnter={onEnter}>
+          <SearchInput
+            ref={searchRef}
+            placeholder="Search the HPE Design System"
+            value={query}
+            onChange={onChange}
+            size="xlarge"
+          />
+        </Keyboard>
+      </Box>
+      <Box overflow="auto" pad="xsmall">
+        <List
+          data={results}
+          onClickItem={onSelect}
+          border={{ side: 'bottom', color: 'border-weak' }}
+          // paginate={results.length > step ? { page } : false}
```

Comment:
Related to [this comment](https://github.com/grommet/hpe-design-system/pull/1871#issuecomment-923373062). 

This should be removed. It was exploring an approach to address the 'blank page' mentioned above. 

This needs more exploration as Pagination within a List is giving some unexpected results when List's data is dynamic. We may need to implement as a controlled pagination instead of using that supplied by List.

Reply:
Thanks for clarifying